en/micro-integrator/docs/references/connectors/google-firebase-connector/google-firebase-connector-example.md

```diff
@@ -0,0 +1,273 @@
+            <property expression="json-eval($.webPushNotificationBody)" name="webPushNotificationBody" scope="default" type="STRING"/>
+            <property expression="json-eval($.webPushNotificationIcon)" name="webPushNotificationIcon" scope="default" type="STRING"/>
+            <property expression="json-eval($.webPushNotificationBadge)" name="webPushNotificationBadge" scope="default" type="STRING"/>
+            <property expression="json-eval($.webPushNotificationImage)" name="webPushNotificationImage" scope="default" type="STRING"/>
+            <property expression="json-eval($.webPushNotificationLanguage)" name="webPushNotificationLanguage" scope="default" type="STRING"/>
+            <property expression="json-eval($.webPushNotificationTag)" name="webPushNotificationTag" scope="default" type="STRING"/>
+            <property expression="json-eval($.webPushNotificationDirection)" name="webPushNotificationDirection" scope="default" type="STRING"/>
+            <property expression="json-eval($.webPushNotificationRenotify)" name="webPushNotificationRenotify" scope="default" type="STRING"/>
+            <property expression="json-eval($.webPushNotificationInteraction)" name="webPushNotificationInteraction" scope="default" type="STRING"/>
+            <property expression="json-eval($.webPushNotificationSilent)" name="webPushNotificationSilent" scope="default" type="STRING"/>
+        </propertyGroup>
+    </sequence>
+   ```
+
+> **NOTE:**
+The parameters under `<init>` section of the configuration above are referring to the credentials we obtained from Google Firebase in above steps. The parameters are mapped to the keys of the Json file that you have downloaded as below. 
+
+```
+accountType --> type
+projectId --> project_id
+privateKeyId --> private_key_id
+privateKey --> private_key
+clientEmail --> client_email
+clientId --> client_id
+authUri --> auth_uri
+tokenUri --> token_uri
+authProviderCertUrl --> auth_provider_x509_cert_url
+clientCertUrl --> client_x509_cert_url
+```
+
+Now we can export the imported connector, sequence and the API into a single CAR application. CAR application is the one we are going to deploy to server runtime.
+
+
+{!references/connectors/exporting-artifacts.md!}
+
+
+Now the exported CApp can be deployed in Enterprise Integrator Runtime so that we can run it and test. 
+
+**NOTE**
+Download following jars. 
+1. [firebase-admin-6.5.0.jar](https://mvnrepository.com/artifact/com.google.firebase/firebase-admin/6.5.0)
+2. [google-auth-library-credentials-0.11.0.jar](https://mvnrepository.com/artifact/com.google.auth/google-auth-library-credentials/0.11.0)
+3. [google-auth-library-oauth2-http-0.11.0.jar](https://mvnrepository.com/artifact/com.google.auth/google-auth-library-oauth2-http/0.11.0)
+4. [api-common-1.7.0.jar](https://mvnrepository.com/artifact/com.google.api/api-common/1.7.0)
+
+and place those into  `<Product_HOME>/lib` folder.
```

Comment:
Merge this with the above line "Download following .jars.."

Reply:
Updated.